src/console/plugin.ts

```diff
@@ -200,9 +200,9 @@ function activateConsole(app: Application, services: ServiceManager, rendermime:
 
   let newSubmenu = new Menu ([
     new MenuItem ({
-      text: 'Python 3 Console',
+      text: services.kernelspecs.default,
```

Comment:
The text should be the same as the one we use elsewhere in this file to create the command palette text.